packages/lambdas/build-shared-layer.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+main() {
+   mkdir -p ./layers/shared/nodejs
+
+   cp -r node_modules ./layers/shared/nodejs/
+
+   core="../core"
+   core_on_layer="./layers/shared/nodejs/node_modules/@metriport/core"
+   # TODO 1148 this can become a function when we want to move all lambda dependencies to local packages to local filesystem instead of NPM
+   mkdir $core_on_layer
+   cp $core/package.json $core_on_layer
+   rsync -a $core/dist $core_on_layer --exclude __tests__
+
+   pushd ./layers/shared
+
+   zip -rq shared-layer.zip ./nodejs
+
+   popd
```

Comment:
nit: this isn't needed afaik

Reply:
since we changed the folder on [line 15](https://github.com/metriport/metriport/pull/1054/files/42a049308329b305b61f8bac750f7723e184099d#diff-215880293427b72ace0a0f5c429d0cc23206910493e18dd2dd3110fc91ff8b55R15), I think we should revert to the previous one so whoever is calling this script will be on the expected folder when this script finishes running.